src/Spectre.libWavelet.Tests/Spectre.libWavelet.Tests.vcxproj

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\opencv3.3.0.0.1\build\native\opencv3.props" Condition="Exists('..\packages\opencv3.3.0.0.1\build\native\opencv3.props')" />
```

Comment:
Why do we have OpenCV included in project file, but not in packages?

Reply:
Oh, it was just the leftover from the project stripping. Do you want me to remove it? It'll be added in the next PR.
And to answer the question precisely - I had to modify the project manually to match all the scripts used there, but forgot to add the opencv to the packages.

Reply:
If it will be added soon, we may just add OpenCV to packages, and we're done ;) Just to keep these two in sync.